db/models/cases.js

```diff
@@ -37,31 +37,45 @@ class Service extends BaseService {
     throw new Error('Could not publish the case. Make sure all are moved into staging state.')
   }
 
+  /**
+   * Consent To Publish
+   *
+   * @method consentToPublishing
+   * @param {Number} case_id
+   * @return {Object}
+   */
+  async consentToPublishing(case_id) {
+    const result = await this.updateOne(case_id, { consented_to_publishing_at: new Date() });
```

Comment:
Also need to add `consented_to_publishing_at -> consentedToPublishingAt` mapping in `_mapCase`. If it's not supposed to be exposed in the returned object, should still `delete itm.consented_to_publishing_at;`.